Doc/library/_thread.rst

```diff
@@ -61,15 +61,28 @@ This module defines the following constants and functions:
       :func:`sys.unraisablehook` is now used to handle unhandled exceptions.
 
 
-.. function:: interrupt_main()
+.. function:: interrupt_main([signum])
 
-   Simulate the effect of a :data:`signal.SIGINT` signal arriving in the main
-   thread. A thread can use this function to interrupt the main thread.
+   Simulate the effect of a signal arriving in the main thread.
+   A thread can use this function to interrupt the main thread.
```

Comment:
Suggestion: Maybe we should add here a small sentence indicating that the signal will be processed by the main thread in the eval loop or when it calls `PyErr_CheckSignals()`.

Reply:
I tweaked the wording slightly. I'm not sure it's good idea to go into a full-length explanation of what happens under the hood, though (especially as it's a CPython implementation detail).